README.md

```diff
@@ -587,7 +587,7 @@ require('go').setup({
                    -- false: do nothing
                    -- if lsp_cfg is a table, merge table with with non-default gopls setup in go/lsp.lua, e.g.
                    --   lsp_cfg = {settings={gopls={matcher='CaseInsensitive', ['local'] = 'your_local_module_path', gofumpt = true }}}
-  lsp_gofumpt = false, -- true: set default gofmt in gopls format to gofumpt
+  lsp_gofumpt = true, -- true: set default gofmt in gopls format to gofumpt
```

Comment:
the comment maybe confusing. But the default value of gofumpt should be `false`.  And please check https://github.com/golang/tools/blob/master/gopls/doc/settings.md.
Maybe change the comments to:
-- if we should run gofumpt formatting.

Reply:
Why by default gofumpt shoud be false? If you run `require('go.format').goimport()` without this patch and with the default settings, then it is formatted using gofumpt.

Reply:
the default format should be gofmt. Same as default format setup of gopls.
If goimport uses gofumpt, then it might be a bug and it should also use gofmt.
Suppose you work in a team, It is more safe to format code with gofmt as gofumpt may cause lots of turmoil.

Reply:
Ok, so I think we should change this
https://github.com/ray-x/go.nvim/blob/a9b669772f131a3000da919f8b9aff64adaf1fe0/lua/go/gopls.lua#L267
to false.

Reply:
yes, this is the place we need to fix;
it should be `gofumpt = _GO_NVIM_CFG.lsp_gofumpt or false`,